integration/travis/scheduler_config.edn

```diff
@@ -19,7 +19,8 @@
          :leader-path "/cook-scheduler"
          :role "cook"
          :framework-id #config/env "COOK_FRAMEWORK_ID"}
- :executor {:command #config/env "COOK_EXECUTOR_COMMAND"}
+ :executor {:command #config/env "COOK_EXECUTOR_COMMAND"
```

Comment:
Should this be `:cook-executor` instead of just `:executor`? These configuration options (especially `:portion`) only apply if you're using the cook executor, correct? Maybe that should be more obvious from the name